app.js

```diff
@@ -0,0 +1,16 @@
+const express = require('express');
+const {getTopics} = require('./controllers/getTopics.controllers')
+
+const app = express()
+
+app.get('/api/topics', getTopics)
+
+app.all('*', (req, res) => {
+    res.status(404).send({ msg: "Not Found" });
+  });
+
+  app.use((err, req, res, next) => {
+    res.status(500).send({ msg: "Internal Server Error" });
+  });
+  
```

Comment:
This is nice to see here just as something to fall back to incase something unexpected goes wrong.